### PR TITLE
c-blosc2: new version, v2.0.0rc1

### PR DESCRIPTION
**Pull request by robert-mijakovic**

### CODE REVIEW

File: var/spack/repos/builtin/packages/c-blosc2/package.py [hunk 1]
```diff
@@ -11,11 +11,13 @@ class CBlosc2(CMakePackage):
        other bells and whistles"""
 
     homepage = "http://www.blosc.org"
+    url      = "https://github.com/Blosc/c-blosc2/archive/refs/tags/v2.0.0.rc1.tar.gz"
     git      = "https://github.com/Blosc/c-blosc2.git"
 
     maintainers = ['ax3l']
 
     version('develop', branch='master')
+    version('2.0.0', sha256='c30b72af5446f052bad1791000e5a44d156c96b0e39b4bc8e9f97a013c7d1b69')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/c-blosc2/package.py]: This is not `v2.0.0` but `v2.0.0-rc1` I assume that it's planned to release the major version later.

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/c-blosc2/package.py (hunk 1 above)
Comment (robert-mijakovic) [reply var/spack/repos/builtin/packages/c-blosc2/package.py]: Correct. Given that it needed 5 years, 8th of Jan  2016, from alpha 2 to RC1, I would expect it in at least a year or two. Therefore, I have included it here as there was just git.

Comment (alalazo) [reply var/spack/repos/builtin/packages/c-blosc2/package.py]: Fair enough, even though `beta5` and `rc1` are just 2 weeks apart. I'm fine with whatever @ax3l thinks is fine with respect to the version tag.

Comment (robert-mijakovic) [reply var/spack/repos/builtin/packages/c-blosc2/package.py]: Nah, you saw it wrongly...  It is 1 year and 2 weeks. :)

Comment (alalazo) [reply var/spack/repos/builtin/packages/c-blosc2/package.py]: :laughing: Correct, I missed that detail.

Comment (ax3l) [reply var/spack/repos/builtin/packages/c-blosc2/package.py]: Yep, I know the RC is open for a bit but that does not mean there won't be a full release :)
Let's name it `2.0.0.rc1`? :-)

Comment (robert-mijakovic) [reply var/spack/repos/builtin/packages/c-blosc2/package.py]: Fixed :)

**Review by haampie — APPROVED** [review]

**Review by alalazo — APPROVED** [review]

**Comment by ax3l** [comment]
blosc2 2.0.0 is officially released :tada: 
https://github.com/Blosc/c-blosc2/releases/tag/v2.0.0

@robert-mijakovic do you like to update the version in this package and co-maintain it? :)

**Comment by robert-mijakovic** [comment]
> blosc2 2.0.0 is officially released 🎉
> https://github.com/Blosc/c-blosc2/releases/tag/v2.0.0
> 
> @robert-mijakovic do you like to update the version in this package and co-maintain it? :)

Sure, I can. :)

**Comment by ax3l** [comment]
X-ref: #24581 :)

**Pull request merged**